FindVitis.cmake

```diff
@@ -75,15 +75,15 @@ endif()
 #if Xilinx tools correctly installed they are added to $ENV{PATH} one of CMake's default search paths
 
 # Find v++
-find_program(VITIS_VPP v++)
+find_program(VITIS_VPP v++ HINTS "${VITIS_ROOT}/bin")
```

Comment:
I don't think this hint should be necessary, if I understand how find_program works?

Reply:
Doesn't work for me without the hint but it's possible that's because I'm setting up `PATH` (but it's a "hint" so what's the harm?).

Reply:
I think when VITIS_ROOT is specified, it's automatically used as a hint, but it may depend on the version of cmake you have. (from https://cmake.org/cmake/help/latest/command/find_program.html)

"<prefix>/[s]bin for each <prefix> in the [<PackageName>_ROOT](https://cmake.org/cmake/help/latest/variable/PackageName_ROOT.html#variable:%3CPackageName%3E_ROOT) CMake variable and the [<PackageName>_ROOT](https://cmake.org/cmake/help/latest/envvar/PackageName_ROOT.html#envvar:%3CPackageName%3E_ROOT) environment variable if called from within a find module loaded by [find_package(<PackageName>)](https://cmake.org/cmake/help/latest/command/find_package.html#command:find_package)"

Reply:
@stephenneuendorffer 

1. Setting just `VITIS_ROOT` doesn't fix/help/affect anything; 
2. Adding `/opt/Xilinx/Vitis/2023.2/bin` to my shell `PATH` does enable the current version to successfully find `v++` (with or without setting `VITIS_ROOT`); 
3. Setting just `Vitis_ROOT` does find but then calling `v++` still fails since the second patch line is missing (namely it should be a call to `VITIS_VPP`). 

I would prefer to merge this change in order to not have to fiddle with `PATH` and instead be able to configure using `-DVitis_ROOT` or `-DVITIS_ROOT` (for maximum "regularity"). Note, I have changed `HINTS` to `PATHS` as the doc recommends:

> 4. Search the paths specified by the `HINTS` option. These should be paths computed by system introspection, such as a hint provided by the location of another item already found. Hard-coded guesses should be specified with the `PATHS` option.

Reply:
> I would prefer to merge this change in order to not have to fiddle with `PATH` and instead be able to configure using `-DVitis_ROOT` or `-DVITIS_ROOT` 

+1

Reply:
@fifield turns out in mlir-aie (at least) the problem is the submodule isn't on the latest commit here; I'm bumping now https://github.com/Xilinx/mlir-aie/pull/760 (so afterwards just setting `-DVitis_ROOT` will work).